### PR TITLE
refresh infer article

### PR DESCRIPTION
**Pull request by simonpcouch**
Updates the content on the newer approach to theoretical methods and adds a section on the new multiple regression features! These two sections are mostly just copied from the main infer vignette.

Closes tidymodels/infer#422.🐢

### CODE REVIEW

File: content/learn/statistics/infer/index.Rmarkdown [hunk 1]
```diff
@@ -232,49 +232,108 @@ null_dist %>%
 
 As you can see, 40 hours per week is not contained in this interval, which aligns with our previous conclusion that this finding is significant at the confidence level $\alpha = .05$.
 
-## Theoretical methods
+## Theoretical Methods
```
Comment (juliasilge) [inline content/learn/statistics/infer/index.Rmarkdown]:
```suggestion
## Theoretical methods
```
The whole site uses sentence case for headings.

File: content/learn/statistics/infer/index.Rmarkdown [hunk 2]
```diff
@@ -232,49 +232,108 @@ null_dist %>%
...
 
-The infer package also provides functionality to use theoretical methods for `"Chisq"`, `"F"` and `"t"` test statistics. 
+{infer} also provides functionality to use theoretical methods for `"Chisq"`, `"F"`, `"t"` and `"z"` distributions.
```
Comment (juliasilge) [inline content/learn/statistics/infer/index.Rmarkdown]:
```suggestion
The infer package also provides functionality to use theoretical methods for `"Chisq"`, `"F"`, `"t"` and `"z"` distributions. 
```

Also, we use the [tidyverse style guide](https://style.tidyverse.org/documentation.html?q=package#capitalization-and-full-stops) for packages, functions, etc.

**Comment by simonpcouch** [comment]
Thank you, @juliasilge. :)

File: content/learn/statistics/infer/index.Rmarkdown [hunk 3]
```diff
@@ -234,47 +234,106 @@ As you can see, 40 hours per week is not contained in this interval, which align
 
 ## Theoretical methods
 
-The infer package also provides functionality to use theoretical methods for `"Chisq"`, `"F"` and `"t"` test statistics. 
+The infer package also provides functionality to use theoretical methods for `"Chisq"`, `"F"`, `"t"` and `"z"` distributions. 
 
-Generally, to find a null distribution using theory-based methods, use the same code that you would use to find the null distribution using randomization-based methods, but skip the `generate()` step. For example, if we wanted to find a null distribution for the relationship between age (`age`) and party identification (`partyid`) using randomization, we could write:
+Generally, to find a null distribution using theory-based methods, use the same code that you would use to find the observed statistic elsewhere, replacing calls to `calculate()` with `assume()`. For example, to calculate the observed $t$ statistic (a standardized mean):
```
Comment (hfrick) [inline content/learn/statistics/infer/index.Rmarkdown]: the math `$t$` doesn't render properly in the netlify preview - is that just my browser/OS (firefox, macOS 12.3.1) or do you see that as well?

File: content/learn/statistics/infer/index.Rmarkdown [hunk 4]
```diff
@@ -234,47 +234,106 @@ As you can see, 40 hours per week is not contained in this interval, which align
...
 
-To find the null distribution using theory-based methods, instead, skip the `generate()` step entirely:
+Then, to define a theoretical $t$ distribution, we could write:
 
 ```{r, message = FALSE, warning = FALSE}
-null_f_distn_theoretical <- gss %>%
-   specify(age ~ partyid) %>%
-   hypothesize(null = "independence") %>%
-   calculate(stat = "F")
+# switch out calculate with assume to define a distribution
```
Comment (hfrick) [inline content/learn/statistics/infer/index.Rmarkdown]: do you want to use `calculate()` and `assume()` here or do you want to stay in English?

File: content/learn/statistics/infer/index.Rmarkdown [hunk 5]
```diff
@@ -234,47 +234,106 @@ As you can see, 40 hours per week is not contained in this interval, which align
...
+
+As an example, suppose that we want to fit `hours` worked per week using the respondent `age` and `college` completion status. We could first begin by fitting a linear model to the observed data.
+
+```{r}
+observed_fit <- gss %>%
+  specify(hours ~ age + college) %>%
+  fit()
+```
+
+Now, to generate null distributions for each of these terms, we can fit 1000 models to resamples of the `gss` dataset, where the response `hours` is permuted in each. Note that this code is the same as the above except for the addition of the `hypothesize` and `generate` step.
```
Comment (hfrick) [inline content/learn/statistics/infer/index.Rmarkdown]:
```suggestion
Now, to generate null distributions for each of these terms, we can fit 1000 models to resamples of the `gss` dataset, where the response `hours` is permuted in each. Note that this code is the same as the above except for the addition of the `hypothesize()` and `generate()` step.
```

**Review by hfrick — APPROVED** [review]
Looks good! Two main things:
- A suggestion: there are a few unnamed chunks in there, including chunks that generate plots. I think it would be helpful to name them because currently `unnamed-chunk-5-1.svg` gets a totally different content and if the name changed as well, it would be another clue that this is intentional. 
- A change request: Could you please re-render with the changes in the last commit?

I think it's good to go after the re-render!

File: content/learn/statistics/infer/index.Rmarkdown (hunk 5 above)
Comment (simonpcouch) [reply content/learn/statistics/infer/index.Rmarkdown]: You got it!👍

File: content/learn/statistics/infer/index.Rmarkdown (hunk 4 above)
Comment (simonpcouch) [reply content/learn/statistics/infer/index.Rmarkdown]: Good point—changing.

File: content/learn/statistics/infer/index.Rmarkdown (hunk 3 above)
Comment (simonpcouch) [reply content/learn/statistics/infer/index.Rmarkdown]: Ah, I see that too. Changed `$\alpha$` and `$z$` to just use code formatting as well.

**Comment by simonpcouch** [comment]
Ready for merging from my perspective, @topepo!

**Pull request merged**